libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRouteWaypoint.kt

```diff
@@ -1,8 +1,8 @@
-package com.mapbox.services.android.navigation.v5.navigation
+package com.mapbox.navigation.route
```

Comment:
Wrong package name, should be 
```suggestion
package com.mapbox.navigation.base.route
```

Reply:
nope